.github/dependabot.yml

```diff
@@ -10,3 +10,5 @@ updates:
     reviewers:
       - "@nugit/front-devs"
     target-branch: "develop"
+    commit-message:
+      prefix: "fix:"
```

Comment:
Should this be `chore:`?

Reply:
`chore` will not trigger a new release. But the `umd` build includes dependencies... So we need `fix` in order to trigger minor version upgrade